posthog/api/utils.py

```diff
@@ -364,11 +391,10 @@ def select_event_definitions(fields: str):
 
     # Only return event definitions
     raw_event_definition_fields = ",".join(event_definition_fields)
-    ordering = (
-        "ORDER BY last_seen_at DESC NULLS LAST, query_usage_30_day DESC NULLS LAST, name ASC"
-        if is_enterprise
-        else "ORDER BY name ASC"
+    provided_ordering = (
+        f"{order_UNSAFE} {order_direction} {'NULLS FIRST' if order_direction == 'ASC' else 'NULLS LAST'}"
```

Comment:
`order_UNSAFE` can be empty (and is empty by default), in which case this will error out.